chroot/create_chroot.sh

```diff
@@ -49,19 +65,35 @@ else
     mkdir -p ${CHROOT_GADGETRON_BINARY_DIR}/chroot/chroot-root/gadgetron
     mkdir -p ${CHROOT_GADGETRON_BINARY_DIR}/chroot/chroot-backups
 
-    apt-get install debootstrap -y
-    debootstrap --variant=buildd --arch amd64 trusty ${CHROOT_GADGETRON_BINARY_DIR}/chroot/chroot-root/gadgetron http://gb.archive.ubuntu.com/ubuntu/
+    # try to find chroot base package
+    CHROOT_BASE=`ls -t ${CHROOT_GADGETRON_BINARY_DIR}/chroot/chroot-backups/gadgetron-base*.tar.gz | head -1`
 
+    if [[ -f "${CHROOT_BASE}" ]]; then
+        echo "find gadgetron chroot base package : ${CHROOT_BASE}"
+        tar -xzf ${CHROOT_BASE} -C ${CHROOT_GADGETRON_BINARY_DIR}/chroot
+        sleep 3
+    else       
+        echo "Cannot find gadgetron chroot base package"
```

Comment:
Why are we not just calling the create_chroot_base.sh at this point. This is now duplicated code.